hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/transaction/TransactionCommitMessageTask.java

```diff
@@ -74,7 +74,7 @@ public String getDistributedObjectName() {
 
     @Override
     public String getMethodName() {
-        return null;
+        return "transactionCommit";
```

Comment:
The method name should be commitTransaction IMO. See TransactionContext.commitTransaction

Reply:
changed.